script/Ciel/Day 3/03_01_CIEL03_3_3B.txt

```diff
@@ -378,7 +377,7 @@ Looking at you right now, even I could probably carry you alone."
 [sha:45f5692ebfb2571172ac25710302c095919eec54]{
 -- Page 218, Offset 5413.
 -- 「ちゃんと体をあっためて、夕ごはんを食べて、しっかりするまで帰らないでください。そんな顔をしたまま出て行かれたら、気になって眠れないじゃないですか」
-"Please don't leave until you've properly warmed up, eaten dinner, and gotten yourself together. If you leave with a face like that, I won't be able to sleep because I'm worried about you."
+"Please don't leave until you've properly warmed up, eaten dinner, and gotten yourself composed. If you leave with a face like that, I'll be up all night with worry."
```

Comment:
I prefer "gotten yourself together"

Reply:
"get yourself together" is a very forceful expression and she says this incredibly gently.